app/src/main/java/com/kahuis/repairtool/presentation/repairs/RepairList.kt

```diff
@@ -0,0 +1,9 @@
+package com.kahuis.repairtool.presentation.repairs
+
+object RepairList {
```

Comment:
Since, you're working with a mock it might be time to deprecate or remove this code.